sympy/utilities/lambdify.py

```diff
@@ -450,6 +455,48 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def should_dummify(expr, printer=None):
+    """
+    Returns the boolean value of dummify to be used.
+
+    Parameter
+    ---------
+    expr : Represents the expression for which dummify should be checked
+    printer : Represents the printer which is provided to this function
+
+    Examples
+    ========
+
+    >>> from sympy.utilities.lambdify import should_dummify
+    >>> from sympy.abc import x
+    >>> should_dummify(r'32v2?g#Gmw845h$Wb53wi\phi')
```

Comment:
I'm also in favor of removing support for str-like inputs. So please remove this example